HorizonEditor/include/ComponentDisplays.h

```diff
@@ -59,7 +59,7 @@ namespace Hzn
 	}
 
 	template<>
-	inline void ComponentDisplays::display<TransformComponent>(const GameObject& obj)
+	inline void ComponentDisplays::display<TransformComponent>(GameObject& obj)
```

Comment:
What was the affects of the const

Reply:
If we had const there it wouldn't let change the game object in any way, so it wouldn't me remove the component from the game object